static/src/stylesheets/layout/nav/_cta-bar.scss

```diff
@@ -1,20 +1,4 @@
 .new-header__cta-bar {
-    .hide-support-messaging &,
-    .is-recent-contributor & {
-        display: none;
```

Comment:
css for hiding/showing is no longer required. This decision making happens in SDC